CONTRIBUTING.md

```diff
@@ -1,14 +1,14 @@
-Thank for for considering a contribution! Generally, Geyser welcomes PRs from everyone. There are some guidelines about what features should go where:
+Thank you for considering a contribution! Generally, Geyser welcomes PRs from everyone. There are some guidelines about what features should go where:
 
 
-*Pull requests that may not get accepted:* Niche features that apply to a specific group, for example integration with a specific plugin. For now, please create a separate plugin if possible. 
+*Pull requests that may not get accepted:* Niche features that apply to a specific group, for example, integration with a specific plugin. For now, please create a separate plugin if possible.
 
 *Pull requests for Floodgate:* Anything that opens up information within the game for developers to use.
 
-*Pull requests for Geyser:* Anything that fixes compatibility between Java or Bedrock, or improves the quality of play for Bedrock players. The exception is wherever direct server access is required; in this case it may be better for Floodgate.
+*Pull requests for Geyser:* Anything that fixes compatibility between Java or Bedrock or improves the quality of play for Bedrock players. The exception is wherever direct server access is required; in this case, it may be better for Floodgate.
 
 
-We have some general style guides that should be applied throughout the code:
+We have some general style guides that you should apply throughout the code:
```

Comment:
Seems silly.

Reply:
Well, who's doing the applying? :p

Reply:
Everyone, not just the reader.

Reply:
Well the reader is reading these guidelines, right?

Reply:
We should let them know that we're not just enforcing the guidelines to you, but to everyone! :)

Reply:
But you is everyone!! I don't get it

Reply:
You is only the reader. Commenting code applies to everyone, even if they do not read this document.

Reply:
```suggestion
We have some general style guides that should be applied throughout the code:
```